common/scaffolding/typographie.html

```diff
@@ -0,0 +1,39 @@
+---
+{
+	"title": "Typographie",
+	"language": "fr",
+	"altLangPage": "typography.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-08-24"
+}
+---
+
+<h1>Titres</h1>
+<h1>Il s'agit d'un élément de titre h1</h1>
+<h2>Il s'agit d'un élément de titre h2</h2>
+<h3>Il s'agit d'un élément de titre h3</h3>
+<h4>Il s'agit d'un élément de titre h4</h4>
```

Comment:
Now this page do contains 3 heading level 1 which is problematic,

The "title" set in the front-matter is rendered as an h1 for this page.

Please apply the following correction:

* [ ] change line 13 for an heading level 2 as expected.
* [ ] Add a paragraph explaining (and please rewrite it) where it means the following content are using HTML semantic heading for demoing their rendering and styling. We added an aria role `paragraph` to them to minimize the impact with assistive technology.
* [ ] Apply a change like:

```suggestion

<h2>Titres</h2>

<h1 role="paragraph">Il s'agit d'un élément de titre h1</h1>
<h2 role="paragraph">Il s'agit d'un élément de titre h2</h2>
<h3 role="paragraph">Il s'agit d'un élément de titre h3</h3>
<h4 role="paragraph">Il s'agit d'un élément de titre h4</h4>
```

Reply:
Fixed and addressed in commit "Updated headings and added paragraph".